worthapp/js/src/what-did-we-learn.js

```diff
@@ -0,0 +1,19 @@
+/* global $, initActivityPanels */
+
+(function() {
+    $(document).ready(function() {
+        var $container = $('.container.what-did-we-learn');
+        initActivityPanels($container);
+
+        $container.find('button.get-answers').click(function(e) {
+            e.preventDefault();
+            var $this = $(this);
+            var $panel = $this.closest('.panel');
+            var chosen = $panel.find('input:checked').attr('value');
+
+            $this.closest('.panel').find('.alert').addClass('hidden');
```

Comment:
You can re-use the `$panel` variable here, that you defined above.

Looks great, though, it's working well.